index.d.ts

```diff
@@ -1,3 +1,63 @@
+import { SnakeCase } from "type-fest";
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type EmptyTuple = [];
+
+/**
+Return a default type if input type is nil.
+@template T - Input type.
+@template U - Default type.
+*/
+type WithDefault<T, U extends T> = T extends undefined | void | null ? U : T;
+
+/**
+Check if an element is included in a tuple.
+TODO: Remove this once https://github.com/sindresorhus/type-fest/pull/217 is merged.
+*/
+type IsInclude<List extends readonly unknown[], Target> = List extends undefined
+  ? false
+  : List extends Readonly<EmptyTuple>
+  ? false
+  : List extends readonly [infer First, ...infer Rest]
+  ? First extends Target
+    ? true
+    : IsInclude<Rest, Target>
+  : boolean;
+
+/**
+Append a segment to dot-notation path.
+*/
+type AppendPath<S extends string, Last extends string> = S extends ""
+  ? Last
+  : `${S}.${Last}`;
+
+/**
+Convert keys of an object to camelcase strings.
+*/
+type SnakeCaseKeys<
```

Comment:
This seems to confuse GitHub's syntax highlighting. Not necessarily a problem, just leaving this as a note to play with it a bit and check whether syntax highlighting works in other engines.

Reply:
I think so too.
This may be caused by unexpected new line after `<` but I don't know how to fix, sorry.